src/nvim/terminal.c

```diff
@@ -383,6 +383,7 @@ void terminal_enter(bool process_deferred)
         if (got_bs) {
           goto end;
         }
+        // FALLTHROUGH
```

Comment:
Coverity complains about this, but not the above cases? If so, I suppose because they have not statements...